README.md

```diff
@@ -158,40 +207,66 @@ The notification microservice v1_1 is a notification service that logs to a file
 
 * To allow users to receive tweets, complete the [Twitter Configuration](#twitter-configuration) (This may have already been done when configuring the user microservice)
 
-* Optionally, update the location/name of the administrative log and fallback log by updating the entry located under the
-   "Optional Microservice Configuration" label in the "properties" section of the
+* Optionally, update the location/name of the administrative log and fallback log. For Maven, update the entry located under the "Optional Microservice Configuration" label in the "properties" section of the
    sample-acmegifts/microservice-notification_v1_1 pom.xml. The default values are: 
    
    `notification_1_1.log.file = ${project.build.directory}/logs/${project.build.directory}/logs/notifications_1_1.log`
    
    `notification_1_1.fallback.log.file = ${project.build.directory}/logs/notifications_1_1_fallback.log`   
+   
```

Comment:
For alot of these, you created a subsection for Maven and Gradle. Just wondering why you didn't follow that same pattern here.

Reply:
Hmm, I am not so sure either let me fix that.